test/pkg/testconfig/testconfig.go

```diff
@@ -51,7 +52,7 @@ const (
 	DualNICBoundaryClockString = "DualNICBC"
 	ptp4lEthernet              = "-2"
 	ptp4lEthernetSlave         = "-2 -s"
-	phc2sysGM                  = "-a -r -r -n 24"
+	phc2sysGM                  = "" // do not autoconfigure phc2sys, use local IF clock as clock source for grandmaster
 	phc2sysSlave               = "-a -r -n 24"
```

Comment:
for events you need 
ptp4lOpts: "-2 --summary_interval -4" 
phc2sysOpts: "-a -r -m -n 24 -N 8 -R 16"

Reply:
added the new parameters